components/dashboard/src/settings/SelectIDE.tsx

```diff
@@ -176,20 +177,45 @@ function orderedIdeOptions(ideOptions: IDEOptions) {
         });
 }
 
-function renderIdeOption(option: IDEOption, selected: boolean, onSelect: () => void): JSX.Element {
+function renderIdeOption(
+    option: IDEOption,
+    selected: boolean,
+    version: IDEOption["imageVersion"],
```

Comment:
I don't think we can do it, because JB versions can have or no this prefix already. I think we should go for now as it and then see that we actually will get from JB.

Right now EAP is a released one, so there is no EAP, besides Rider custom handling

Reply:
Sounds good. Let's do this! 🤝